server/src/main/java/org/apache/uniffle/server/storage/LocalStorageManager.java

```diff
@@ -301,24 +302,36 @@ public void removeResources(PurgeEvent event) {
 
   private void cleanupStorageSelectionCache(PurgeEvent event) {
     Function<String, Boolean> deleteConditionFunc = null;
+    String prefixKey = null;
```

Comment:
Please handle the `NPE` of `perfixKey` although event always will be hit. But for the further more implementations of event, I think we should check the NPE